dialects/mem/mem.thorin

```diff
@@ -74,22 +76,34 @@
 ///
 /// ### %mem.lea
 ///
-/// Load effective address. 
+/// Load effective address.
 /// Performs address computation by offsetting the passed pointer with index `i`.
 .ax %mem.lea: Π [n: .Nat, Ts: «n; *», as: .Nat][%mem.Ptr(«j: n; Ts#j», as), i: .Idx n] -> %mem.Ptr(Ts#i, as), normalize_lea;
 ///
+/// ### %mem.create/destroy
+///
+/// Create side effect out of thin or pretend that the side effect does not exists.
+/// @warning Use with caution since you completely remove the `%%mem.M` dependency.
+///
+.ax  %mem.m: %mem.M;
```

Comment:
This is automatically resolved in the already existing memory-polishing phases and as such equivalent to `.Top : %mem.M`?